package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@nsftx/seven-gravity-gateway",
   "private": false,
-  "version": "3.0.3-beta.0",
+  "version": "3.0.2",
```

Comment:
Is this version change correct?

Reply:
Yea. It is just sync with parent. It is little bit off since we introduced beta on 3.0.3 (but it was never released in prod).